.github/workflows/quality.yaml

```diff
@@ -1,24 +1,20 @@
+---
 name: Code quality check
 on:
-  push
+  pull_request:
+    branches:
+      - master
 jobs:
   python_quality_check:
     name: Run Python quality checks
     runs-on: ubuntu-latest
+
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
-      - name: Setup Python
-        uses: actions/setup-python@v4
+      - name: Install Flox
+        uses: flox/install-flox-action@v2
+      - name: Run code quality checks
+        uses: flox/activate-action@v1
         with:
-          python-version: "3.13"
-      - name: Install uv
-        uses: astral-sh/setup-uv@v3
-      - name: Setup Mise
-        uses: jdx/mise-action@v2
-        with:
-          install: true
-          install_args: "-y"
-          experimental: true
-      - name: Run Python quality checks
-        run:  mise tasks run python:lint
+          command: LD_PRELOAD="" mise run lint
```

Comment:
What is `LD_PRELOAD` argument this for?

Reply:
this fixes the issues with flox per flox:

https://x.com/hyperpriorai/status/1915423983987499061